### PR TITLE
Update isort from version 4 to 5

### LINKED ISSUE #4: Flake8 failures in TravisCI
Example: https://travis-ci.org/github/zopefoundation/Products.StandardCacheManagers/jobs/733660534

### PR DESCRIPTION
**Pull request by jugmac00**
fixes #4 

modified:   setup.cfg
modified:   src/Products/StandardCacheManagers/RAMCacheManager.py
modified:   tox.ini

### CODE REVIEW

**Review by dataflake — CHANGES_REQUESTED** [review]
Remove the reference to flake8-isort in tox.ini, it is redundant

**Comment by jugmac00** [comment]
> Remove the reference to flake8-isort in tox.ini, it is redundant

As mentioned in https://github.com/zopefoundation/DocumentTemplate/pull/56#issuecomment-705344682 - is it ok than to enforce the sort order via `isort` again? Ie change the `isort` check from `- isort {toxinidir}/src setup.py` to `- isort {toxinidir}/src setup.py`?

**Comment by dataflake** [comment]
> > Remove the reference to flake8-isort in tox.ini, it is redundant
> 
> As mentioned in [zopefoundation/DocumentTemplate#56 (comment)](https://github.com/zopefoundation/DocumentTemplate/pull/56#issuecomment-705344682) - is it ok than to enforce the sort order via `isort` again? Ie change the `isort` check from `- isort {toxinidir}/src setup.py` to `- isort {toxinidir}/src setup.py`?

Yes

**Comment by dataflake** [comment]
I see the same issue here as with the DocumentTemplate changes. For some reason you remove both `--diff` and ``--check-only``. Please don't do that.

**Comment by jugmac00** [comment]
> I see the same issue here as with the DocumentTemplate changes. For some reason you remove both `--diff` and `--check-only`. Please don't do that.

In this repo there was no `diff` flag used anyway. But I guess finally we figured out what's the best combination of these surprisingly acting flags.

File: tox.ini [hunk 1]
```diff
@@ -43,10 +43,10 @@ deps =
...
     flake8-todo
-    flake8-isort
     mccabe
     flake8-blind-except
     flake8-commas
 commands =
-    - isort --check-only --recursive {toxinidir}/src setup.py
+    isort --check-only --diff {toxinidir}/src setup.py
     flake8 --doctests src setup.py {posargs}
+    isort --version
```
Comment (dataflake) [inline tox.ini:52]: Is this needed?

Comment (jugmac00) [reply tox.ini]: I overlooked this - I needed this for debugging some "special" behaviour of isort.

Removed it.

**Review by dataflake — APPROVED** [review]

**Comment by jugmac00** [comment]
Thanks for your time!

**Pull request merged**